torch/csrc/jit/python/script_init.cpp

```diff
@@ -785,7 +785,8 @@ void initJitScriptBindings(PyObject* module) {
                 try {
                   return toPyObject(self.attr(name));
                 } catch (const ObjectAttributeError& err) {
-                  throw AttributeError("%s", err.what());
+                  pybind11::set_error(PyExc_AttributeError, err.what());
```

Comment:
throw AttributeError wasn't already pybind11's way of signaling an attribute error?

Reply:
it requires some extra boilerplate to actually translate the exception apparently, which I minimally reproduce below.

I did also try `throw pybind11::attribute_error` but for some reason this got translated into a runtime error :/